src/apps/activity_assignments/crud/assignments.py

```diff
@@ -78,38 +79,39 @@ async def create_many(self, schemas: list[ActivityAssigmentSchema]) -> list[Acti
 
         return await self._create_many(schemas)
 
-    async def already_exists(self, schema: ActivityAssigmentSchema) -> ActivityAssigmentSchema:
+    async def exist(self, assignment: ActivityAssignmentCreate) -> ActivityAssigmentSchema | bool:
         """
```

Comment:
Use a Class or a boolean as return not seems right, they are two different concepts, it will never return true, so it should better to use None, seems better to test if (value is not None)